service/src/main/kotlin/io/provenance/explorer/service/ExplorerService.kt

```diff
@@ -116,4 +119,107 @@ class ExplorerService(
         ChainGasFeeCacheRecord.findForDates(fromDate, toDate, count).reversed()
 
     fun getChainId() = asyncCaching.getChainIdString()
-}
+
+    fun getParams() : Params {
```

Comment:
This is exactly how I thought it would come out. Good job getting it all together. I know its a pain >.<